config/worldwide_tagging_organisations.yml

```diff
@@ -0,0 +1,8 @@
+# This file has a list of organisation content_id's
+# that are allowed to have their documents tagged into the worldwide taxonomy
```

Comment:
Very subtle, but I think this could be misread, as this list controls access to the tagging interface.

We're not doing anything about checking the organisations of the content actually tagged to the Worldwide taxons, which is what I think the description here could infer?

Reply:
Perhaps. So what change it to something like: 

```
# This file has a list of organisation content_id's
# that are allowed to access the web interface to tag their documents to the worldwide taxonomy
```

It's the same comment that's in `config/organisations_in_tagging_beta.yml` does that need updating too?

Reply:
Yeah, that looks good. I think the "access the web interface to" could also be omitted, without loosing the important information.

As for the other file, yep, similar changes would be good.